crates/extensions/c8y_auth_proxy/src/server.rs

```diff
@@ -890,7 +892,7 @@ mod tests {
     #[tokio::test]
     async fn forwards_unsuccessful_responses() {
         let _ = env_logger::try_init();
-        let mut server = mockito::Server::new();
+        let mut server = mockito::Server::new_async().await;
         let _mock = server
             .mock("GET", "/not-a-known-url")
             .with_status(404)
```

Comment:
You missed to apply `create_async` to the line below. (line 899)

Reply:
:-) I will fixed that.

That said this demonstrates that @Bravo555 comment is correct. The tests are working as expected with `create()` as well ass with `create_async()`.